### PR TITLE
[spaceship] Update Model: beta_tester

### PR DESCRIPTION
**Pull request by mlch911**
### Checklist

- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I see several green `ci/circleci` builds in the "All checks have passed" section of my PR ([connect CircleCI to GitHub](https://support.circleci.com/hc/en-us/articles/360008097173-Why-aren-t-pull-requests-triggering-jobs-on-my-organization-) if not)
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.
- [x] I've added or updated relevant unit tests.

### Description

Add missing attributes when get BetaTester from ConnectAPI.

### CODE REVIEW

File: spaceship/lib/spaceship/connect_api/models/beta_tester.rb [hunk 1]
```diff
@@ -9,6 +9,14 @@ class BetaTester
       attr_accessor :email
       attr_accessor :invite_type
       attr_accessor :invitation
+      attr_accessor :beta_tester_state
```
Comment (lacostej) [inline spaceship/lib/spaceship/connect_api/models/beta_tester.rb:12]: Thanks for the contribution!

Could you update the unit test in `./spaceship/spec/connect_api/models/beta_tester_spec.rb`?

Thanks

**Review by lacostej — APPROVED** [review]

File: spaceship/lib/spaceship/connect_api/models/beta_tester.rb (hunk 1 above)
Comment (mlch911) [reply spaceship/lib/spaceship/connect_api/models/beta_tester.rb:12]: Does this response in `beta_tester_spec.rb` come from `spaceship/spec/connect_api/fixtures/testflight/beta_testers.json` ?

Comment (lacostej) [reply spaceship/lib/spaceship/connect_api/models/beta_tester.rb:12]: I think so

Comment (mlch911) [reply spaceship/lib/spaceship/connect_api/models/beta_tester.rb:11]: Done. @lacostej

File: spaceship/spec/connect_api/fixtures/testflight/beta_testers.json [hunk 2]
```diff
@@ -7,7 +7,27 @@
...
         "email" : "email@email.com",
         "inviteType" : "EMAIL",
-        "invitation" : null
+        "isDeleted": false,
+        "betaTesterState": "INSTALLED",
+        "lastModifiedDate": "2024-01-21T20:52:18.921-08:00",
+        "installedCfBundleShortVersionString": "1.3.300",
+        "installedCfBundleVersion": "1113",
+        "removeAfterDate": "2024-04-20T00:00:00-07:00",
+        "latestExpiringCfBundleShortVersionString": "1.3.300",
```
Comment (lacostej) [inline spaceship/spec/connect_api/fixtures/testflight/beta_testers.json:16]:
```
       "latestExpiringCfBundleShortVersionString": "1.3.300",
        "latestExpiringCfBundleVersionString": "1113",
        "installedDevicePlatform": "IOS",
        "latestInstalledDevice": "iPhone14_7",
        "latestInstalledOsVersion": "17.2.1",
        "latestInstalledDevicePlatform": "IOS",
```

do we want to add these?

Comment (lacostej) [reply spaceship/spec/connect_api/fixtures/testflight/beta_testers.json:16]: Let's add them and update the test. OK for you @mlch911 ?

Comment (mlch911) [reply spaceship/spec/connect_api/fixtures/testflight/beta_testers.json:16]: No problem.

**Pull request merged**